src/Controls/src/Core/Compatibility/Handlers/Windows/VisualElementRenderer.cs

```diff
@@ -89,9 +89,7 @@ protected override WSize MeasureOverride(global::Windows.Foundation.Size availab
 			{
 				if (child is Maui.IElement childElement && childElement.Handler is IPlatformViewHandler nvh)
 				{
-					var size = nvh.GetDesiredSizeFromHandler(availableSize.Width, availableSize.Height);
-					height = Math.Max(height, size.Height);
-					width = Math.Max(width, size.Width);
+					_ = nvh.GetDesiredSizeFromHandler(width, height);
```

Comment:
This change only effects `FrameRenderer`. None of our other classes (ListView, TableView) that inherit from `VisualElementRenderer` have `LogicalChildren`

Reply:
Can you add this as a comment in the code too?

Reply:
I backed these changes out of this PR. The PR now only has changes inside `FrameRenderer`